tools/query_breakdown/src/main/java/com/google/bigquery/InputReader.java

```diff
@@ -12,31 +11,62 @@
  */
 public class InputReader {
 
+  private LocationTracker lt;
```

Comment:
Just use name `locationTracker`.

Reply:
should I change all the lt's to locationTracker?

Reply:
yes

Reply:
fixed!